sklearn/model_selection/tests/test_search.py

```diff
@@ -1710,19 +1710,23 @@ def test_deprecated_grid_search_iid():
     depr_message = ("The default of the `iid` parameter will change from True "
                     "to False in version 0.22")
     X, y = make_blobs(n_samples=54, random_state=0, centers=2)
-    grid = GridSearchCV(SVC(gamma='scale'), param_grid={'C': [1]}, cv=3)
+    grid = GridSearchCV(SVC(gamma='scale', random_state=0),
```

Comment:
We do not always warn.
https://github.com/scikit-learn/scikit-learn/blob/d19a5dcb0444146f0e4c8c444f410b290dcd9b41/sklearn/model_selection/_search.py#L795-L813